internal/metrics/census.go

```diff
@@ -118,7 +118,7 @@ func InitCensus(nodeID, version, namespace string) {
 	baseTags := []tag.Key{Census.kNodeID}
 	views := []*view.View{
 		{
-			Name:        "versions",
+			Name:        "streamtester_versions",
```

Comment:
Will this have any relevant effect the other binaries built on this repository?

Reply:
Yes, but I think we don't use this metric anywhere.